pkgs/development/python-modules/graphviz/default.nix

```diff
@@ -14,7 +18,20 @@ buildPythonPackage rec {
     sha256 = "d311be4fddfe832a56986ac5e1d6e8715d7fcb0208560da79d1bb0f72abef41f";
   };
 
-  propagatedBuildInputs = [ pkgs.graphviz ];
+  postPatch = ''
+    substituteInPlace graphviz/backend.py \
+      --replace "cmd = [" "cmd = ['${graphviz}/bin/' + "
+    substituteInPlace tests/test_backend.py \
+      --replace "assert_called_once_with(['dot'" "assert_called_once_with(['${graphviz}/bin/dot'" \
+      --replace "@pytest.mark.usefixtures('empty_path')" "@pytest.mark.skip(reason='empty $PATH has no effect')" \
+      --replace "assert capsys.readouterr() == (''', ''')" "" # Fontconfig error: cannot load default config file
```

Comment:
@FRidh Would you prefer doing this stuff with a proper patch?

Reply:
Patches are typically better because they clearly fail in case of changes. Aside from that, I suppose this is fine as well.

Reply:
It might be easier to skip those tests with `pytest -k`. We have examples for that in nixpkgs.

Reply:
If I'm providing a patch anyway, I don't see any advantage in using `-k`. I'll switch to using a patch in the next days.

Reply:
See https://github.com/NixOS/nixpkgs/blob/3b19fa97455081f26c675cf001e8decc5f9c14ca/pkgs/development/libraries/pango/default.nix#L31-L34 for the fontconfig error